src/client/Microsoft.Identity.Client/TelemetryCore/AssertionType.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client.TelemetryCore
+{
+    internal enum AssertionType
+    {
+        Certificate,
```

Comment:
Please add `None` as default value and assign int values to each member + update the doc (add an Annex). Folks creating Kusto queries based on this data will need to know what the numbers mean.